### ISSUE TITLE
Editing schedules wipes out script parameters

### ISSUE THREAD

**Issue by wdomon** [ironmansoftware/powershell-universal#2331]
### Version

3.7.14

### Severity

High

### Steps to Reproduce

1. Create a schedule that runs a script, entering values for script parameters.
2. Refresh the "Schedules" page at least one time (bug not present until page is reloaded)
3. Click "Edit" on above schedule
4. All script parameters fields are now missing from the edit dialog/modal
5. Choose a new date/time and save changes
6. Expand schedule (or view in schedules.ps1) to find that all parameters are now empty strings
7. Close edit dialog
8. Click "Edit" a second time (before a page refresh) and parameter fields are now no longer missing from the dialog

### Expected behavior

```console
Edit Schedule dialog should contain parameter fields on first click but also they should A) contain the values originally entered when the schedule was created (this should also be true for the date/time), and B) Parameter field names should respect [ComponentModel.DisplayName()] like scripts do. Currently, the Edit Schedule dialog always shows the parameter as it is written in the script rather than a displayname while the Invoke Script dialog shows a friendly displayname when [ComponentModel.DisplayName()] is used.
```

### Actual behavior

```console
Edit schedule dialog is missing parameter fields entirely when first opened. This makes it seem like the parameters are not being changed and only the properties listed are what are being changed. If a new date/time is selected and the change is saved, all script parameters that were previously set are wiped out and set as empty strings in the Schedules interface as well as in the schedules.ps1 file; resulting in script failure, etc. If Edit is clicked again before the page is refreshed, the parameter fields are present but empty, requiring the values to be entered again or be replaces with empty strings.
```

### Additional Environment data

OS: Server 2022
PSU: 3.7.14
Confirmed in browsers: Brave, Chrome, Chromium Edge
(Can provide screenshots if needed but seems to be easy enough to reproduce)

**Comment by adamdriscoll** [comment]
This was resolved in 3.8.4: https://docs.powershelluniversal.com/changelog#3.8.4-3-26-2023

**Comment by wdomon** [comment]
Thanks, I had looked for a closed issue and I guess missed this.
